### PR TITLE
[bitrise] Added new build service Bitrise.io

### PR DESCRIPTION
**Pull request by freezy-sk**
Added new service Bitrise.io requested in #574

### CODE REVIEW

File: service-tests/bitrise.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'bitrise', title: 'Bitrise' });
+module.exports = t;
+
+t.create('deploy status')
```
Comment (paulmelnikow) [inline service-tests/bitrise.js]: Since the branch is optional, could you add a second test which omits it?

**Comment by paulmelnikow** [comment]
Thanks! Good to merge when tests are passing. It looks like there is a syntax error in Node 6.

**Comment by freezy-sk** [comment]
@paulmelnikow sorry I forgot commas :)

**Comment by paulmelnikow** [comment]
Thanks!

**Pull request merged**